src/main/java/com/couchbase/intellij/tree/cblite/CBLDataLoader.java

```diff
@@ -59,6 +59,29 @@ public static SavedCBLDatabase saveNewDatabase(String id, String name, String pa
         return newdDb;
     }
 
+    public static void listScopes(DefaultMutableTreeNode parentNode) throws CouchbaseLiteException {
```

Comment:
Never **ever** commit code that is not being used.

Reply:
okay. removing it.